.pre-commit-config.yaml

```diff
@@ -17,7 +17,7 @@ repos:
     - id: isort
       args:
         - --settings-path=.github/linters/pyproject.toml
-        - --check
+#        - --check
```

Comment:
revert it?

Reply:
Maybe we can just keep it as it is? It is both OKAY to keep it or not in online CI, but it will be more convenient to comment it in offline :)